kubernetes/kube-state-metrics-deployment.yaml

```diff
@@ -1,5 +1,5 @@
-apiVersion: apps/v1beta2
-# Kubernetes versions after 1.9.0 should use apps/v1
+apiVersion: apps/v1
+# Kubernetes version 1.8.x should use apps/v1beta2
```

Comment:
```suggestion
# Kubernetes versions after 1.8.x should use apps/v1beta2
```

Reply:
> Kubernetes versions after 1.8.x should use apps/v1beta2

Doesn't that imply that versions 1.8 and higher should use `apps/v1beta2`, when it's really only the 1.8 series? Versions 1.9+ should use (the default) `apps/v1`.